micro-deps/micro-deps-spring-config/src/test/groovy/com/ofg/infrastructure/discovery/ApplicationContextStartupSpec.groovy

```diff
@@ -27,6 +28,11 @@ class ApplicationContextStartupSpec extends Specification {
             applicationContext?.close()
             testingServer?.close()
     }
-         
+
+    TestingServer startedZooKeeper() {
```

Comment:
Do we have more places where that method be useful? Maybe there could be some static method(s) in `AvailablePortScanner` for "standard" cases?